internal/services/config/config.go

```diff
@@ -64,6 +64,8 @@ type Gateway struct {
 	TokenSigning TokenSigning `yaml:"tokenSigning"`
 
 	AdminToken string `yaml:"adminToken"`
+
+	EnableInvitation bool `yaml:"enableInvitation"`
```

Comment:
I don't think EnableInvitation as is makes much sense. I was thinking about an org option to enable disable invitations. A global option should be declared to define the default behavior for new organizations (`NewOrgInvitiationEnabled`).